plate_mapper/plate_mapper.py

```diff
@@ -7,20 +7,48 @@
 # ----------------------------------------------------------------------------
 
 
+import sys
 import argparse
 
 
-def plate_mapper(input_f, barseq_f, output_f):
+def _print_list(l):
+    """ Print a list of strings in a human-friendly way
+
+    Parameter
+    ---------
+    l : list of str
+        Input list of strings
+
+    Return
+    ------
+    str
+        Output string
+
+    Notes
+    -----
+    If the list has <= 10 items, the entire list will be printed.
```

Comment:
Hmm... I feel it is not necessary to add output message customization at the cost of adding another command-line parameter. The user does not need to see the whole list if there are many. If the first few doesn't look right to them, they will go back to check what was wrong. How does that sound?